R/intercept.R

```diff
@@ -1,11 +1,10 @@
 #' Add intercept (or constant) column
 #'
-#' `step_intercept` creates a *specification* of a recipe step that
-#'   will add an intercept or constant term in the first column of a data
-#'   matrix. `step_intercept` has defaults to *predictor* role so
-#'   that it is by default called in the bake step. Be careful to avoid
-#'   unintentional transformations when calling steps with
-#'   `all_predictors`.
+#' `step_intercept()` creates a *specification* of a recipe step that will add
+#' an intercept or constant term in the first column of a data matrix.
+#' `step_intercept()` has defaults to *predictor* role so that it is by default
+#' called in the bake step. Be careful to avoid unintentional transformations
```

Comment:
```suggestion
#' only called in the bake step. Be careful to avoid unintentional transformations
```

Is this what this sentence means?